app/models/vic/profile_photo_attachment.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module VIC
+  class ProfilePhotoAttachment < FormAttachment
+    ATTACHMENT_UPLOADER_CLASS = ProfilePhotoAttachmentUploader
+
+    def set_file_data!(file, in_progress_form)
+      attachment_uploader = ProfilePhotoAttachmentUploader.new(SecureRandom.hex(32), in_progress_form)
+      attachment_uploader.store!(file)
+
+      file_data = {
+        filename: attachment_uploader.filename,
+        path: attachment_uploader.store_dir,
+        user_uuid: in_progress_form&.user_uuid
+      }.compact
+
+      self.file_data = file_data.to_json
+    end
```

Comment:
can you override get_file_data so that it will work with the new storage scheme?

Reply:
Yep, forgot about that. Thanks.